rootfs/init.poplar.rc

```diff
@@ -27,3 +27,14 @@ service hiavplayer /system/vendor/bin/hiavplayer
     class main
     user  system
     group audio
+
+on post-fs-data
+    # create for OP-TEE test
+    mkdir /data/tee 0775 shell shell
+    mkdir /data/tee/optee_armtz 0775 shell shell
+    restorecon_recursive /data/tee
+
+service tee-supplicant /system/bin/tee-supplicant
+    class main
+    user root
```

Comment:
does the user has to be root?

Reply:
We discussed this at connect hacking with SWG guys. The possibility of having a tee user, or maybe switching to mediadrm user as that is what drm plugins run as. I'd be interested what trusty is doing. I think currently most people have been running xtest as root.